<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestTouchScreenPage</class>
 <widget class="QWidget" name="TestTouchScreenPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="ICTouchScreenTestWidget" name="touchTestWidget" native="true">
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>520</y>
        <width>97</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="passButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>520</y>
        <width>97</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>通过</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rejectButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>520</y>
        <width>97</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>不通过</string>
      </property>
     </widget>
     <widget class="QLabel" name="infoLabel">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>490</y>
        <width>461</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>请在屏幕上随意触摸滑动，观察光标是否跳动</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICTouchScreenTestWidget</class>
   <extends>QWidget</extends>
   <header>ictouchscreentestwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
